RSS Reader/RSS Reader/Topics list/View /RSSRTopicsListTableViewController.m

```diff
@@ -0,0 +1,127 @@
+//
+//  RSSRTopicsListTableViewController.m
+//  RSS Reader
+//
+//  Created by Lina Loyko on 11/17/20.
+//
+
+#import "RSSRTopicsListTableViewController.h"
+#import "RSSRTopic.h"
+#import "RSSRTopicsListPresenter.h"
+#import "RSSRTopicTableViewCell.h"
+#import "RSSRNetworkService.h"
+#import "RSSRXMLParser.h"
+
+static NSString * const kReuseIdentifier = @"RSSRTopicTableViewCell";
+static NSString * const kTitle = @"TUT.by News";
+
+@interface RSSRTopicsListTableViewController ()
+
+@property (nonatomic, retain) RSSRTopicsListPresenter *presenter;
+@property (nonatomic, retain) NSMutableArray<RSSRTopic *> *dataSource;
+
+@end
+
+@implementation RSSRTopicsListTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.tableView.separatorStyle = UITableViewCellSeparatorStyleNone;
+    self.tableView.backgroundColor = UIColor.whiteColor;
+    self.title = kTitle;
+    self.navigationController.title = self.title;
+    self.navigationController.navigationBar.translucent = NO;
+    [self.navigationController setHidesBarsOnSwipe:YES];
+    
+    [self.tableView registerNib:[UINib nibWithNibName:kReuseIdentifier bundle:nil] forCellReuseIdentifier:kReuseIdentifier];
+    
+    
+    RSSRXMLParser *parser = [RSSRXMLParser new];
+    RSSRNetworkService *service = [RSSRNetworkService new];
+    RSSRTopicsListPresenter *presenter = [[RSSRTopicsListPresenter alloc] initWithService:service
```

Comment:
keep the view controller as stupid as possible
this means that the view controller should handle UI events only
the whole business logic should be done in objects like presenter 
initialize presenter, for example, within the appdelegate, and there assign initialized and configured presenter to this view controller